src/Autoloader.php

```diff
@@ -1,7 +1,7 @@
 <?php
-spl_autoload_register(function ($class) {
-    if (!preg_match('~Embera~i', $class)) {
-        return ;
+spl_autoload_register(static function ($class) {
+    if (false === stripos($class, 'Embera')) {
```

Comment:
but why yoda condition? lol

Reply:
And why not? I can easily change this in case its required